core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1577,6 +1577,13 @@ private static RelDataType arrayCompactReturnType(SqlOperatorBinding opBinding)
           ReturnTypes.BOOLEAN_NULLABLE,
           OperandTypes.ARRAY_ELEMENT);
 
+  /** The "ARRAY_CONTAINS(element, array)" function. */
+  @LibraryOperator(libraries = {SNOWFLAKE})
+  public static final SqlFunction ARRAY_CONTAINS_SF =
```

Comment:
Pleae add the test case for the `ARRAY_CONTAINS_SF`,  `TRY_PARSE_JSON` and `PARSE_JSON_SF` Sql functions

Reply:
yes, added the tests